browser/model/jbds.js

```diff
@@ -18,8 +18,8 @@ import Util from './helpers/util';
 class JbdsInstall extends InstallableItem {
   constructor(installerDataSvc, downloadUrl, installFile, targetFolderName) {
     super('jbds',
-          'RED HAT JBOSS DEVELOPER STUDIO',
-          '9.1.0',
+          'RED HAT DEVELOPER STUDIO',
```

Comment:
This is the only component with name in upper case, is there any reason for that?